calculator.js

```diff
@@ -0,0 +1,78 @@
+window.onload = function () {
+  let items = [], sumTotal = 0; 
+  function calculate() {
+    function thousandSeperator(n) {
+      let count = 0, numArray = n.split('');
+      for(let i = numArray.length -1; i > 0; i--) {
+        if (count === 2) {
+          numArray.splice(i, 0, ',');
+          count = 0;
+          continue;
+        }
+        count += 1;
+      }
+      return numArray.join('');
+    }
+    let table = document.getElementById('itemTable');
+    let itemName = document.getElementById('itemName').value;
+    let itemQty = document.getElementById('itemQty').value;
+    let itemPrice = document.getElementById('itemPrice').value;
+    if (itemName === '' || itemName.match(' ')) {
```

Comment:
if i input macbook pro as an item name, it reads the input as null, i think the error is coming from this if statement. fix that

Reply:
okay, will look into that